java-base/image.yaml

```diff
@@ -0,0 +1,45 @@
+schema_version: 1
+
+name: redhat-amqstreams-1/amqstreams10-javabase-openshift
+description: "AMQ Streams Java base image"
+version: "1.0"
+from: jboss/openjdk18-rhel7:1.1
+
+labels:
+  - name: "com.redhat.component"
+    value: "amqstreams10-javabase-openshift-container"
+  - name: "io.k8s.description"
+    value: "Base image for Java base"
```

Comment:
Base image for Java base sounds strange :) maybe Base image for Java based components?